Foundation/NSTask.swift

```diff
@@ -7,6 +7,7 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
+#if !os(Android) // not available
```

Comment:
@parkera  is having `@available(Android, unavailable)` a blocker for this PR? IMHO it would be the ideal solution here.